smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/RuntimeClientPlugin.java

```diff
@@ -481,6 +503,21 @@ public Builder pluginFunction(SymbolReference pluginFunction) {
             return this;
         }
 
+        /**
+         * Sets a function symbol reference used to configure clients and
+         * commands to use a specific middleware function.
+         *
+         * @param pluginFunction Plugin function symbol to invoke.
+         * @param additionalParameters Additional parameters to be generated as plugin function input.
+         * @return Returns the builder.
+         * @see #getPluginFunction()
+         */
+        public Builder pluginFunction(SymbolReference pluginFunction, String... additionalParameters) {
```

Comment:
What do additionalParameters contain? Just the names? Or does it also contain the types?

Reply:
The additionalParameters contains just the static names as of now. The types are defined in the actual Plugin implementations.